cnf-certification-test/platform/suite.go

```diff
@@ -166,7 +166,7 @@ func testContainersFsDiff(env *provider.TestEnvironment) {
 	var badContainers []string
 	var errContainers []string
 	for _, cut := range env.Containers {
-		logrus.Debug(fmt.Sprintf("%s(%s) should not install new packages after starting", cut.Podname, cut.Data.Name))
+		logrus.Debug(fmt.Sprintf("%s should not install new packages after starting", cut.String()))
```

Comment:
I think it is more clear to have pod name and container name.

Reply:
This prints all of that via String(): https://github.com/test-network-function/cnf-certification-test/blob/main/pkg/provider/provider.go#L418